templates/home.html

```diff
@@ -44,7 +44,7 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
         {% bootstrap_icon "globe" %}
         <span class="text">
             Request Map<br/>
-            <span class="ml small"></span>
+            <span class="ml small">സഹായം ആവശ്യമായ സ്ഥലങ്ങൾ</span>
```

Comment:
Typo. സ്‌ഥലങ്ങൾ

Reply:
It's just the Github preview. Comes out right in the app.
![image](https://user-images.githubusercontent.com/1825878/44230693-7e03cc80-a1b9-11e8-90fc-bcf0e08ffa9a.png)